pyo3-macros-backend/src/deprecations.rs

```diff
@@ -1,6 +1,7 @@
 use proc_macro2::{Span, TokenStream};
 use quote::{quote_spanned, ToTokens};
 
+#[derive(Clone)]
```

Comment:
Any reason why the `Clone` impl was necessary here and below? I wonder if the changes in this file can be reverted...

Reply:
Good catch! I reverted this change and things still seem to work.